src/main/java/uk/gov/hmcts/ccd/domain/types/DocumentValidator.java

```diff
@@ -58,7 +55,7 @@ public BaseType getType() {
         }
 
         final String documentUrlValue = documentUrl.textValue();
-        final String urlPatternString = applicationParams.getValidDMDomain() + "/documents/[A-Za-z0-9-]+(?:/binary)?";
+        final String urlPatternString = "http.*/documents/[A-Za-z0-9-]+(?:/binary)?";
```

Comment:
Are we confident this cannot be https.*?

Reply:
This may or may not be https. We are tolerating both. Because wherever the document is stored (protocol being part of it by definition) is not our concern any more.